frontend/server/src/DAO/CoderOfTheMonth.php

```diff
@@ -22,8 +22,10 @@ class CoderOfTheMonth extends \OmegaUp\DAO\Base\CoderOfTheMonth {
      */
     public static function calculateCoderOfTheMonth(
         string $startTime,
-        string $endTime
+        string $endTime,
+        string $category = 'all'
     ): ?array {
+        $gender = ($category == 'female') ? " AND i.gender = 'female'" : '';
```

Comment:
nit:

```suggestion
        $genderClause = ($category == 'female') ? " AND i.gender = 'female'" : '';
```

igual en todos los demás métodos.